examples/ipython/tutorial_algebra.ipynb

```diff
@@ -0,0 +1,672 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to using $\\mathcal{G\\!A}$lgebra\n",
```

Comment:
This was perhaps ambitious... It results in a nonsense HTML `<title>` but looks good everywhere else.

Reply:
I can accept the title issue for a pretty `\\mathcal{G\\!A}$lgebra`. I believe the HTML title can be changed back by a hidden javascript if we want.

Reply:
Perhaps I can just use the mathcal parts of Unicode...

Reply:
𝒢𝒜lgebra

Reply:
𝓖𝓐lgebra

Reply:
Removed for now, if we want to play this game it probably makes sense in the logo position in the top corner.

Reply:
ok